bin/tests/integration/in_memory.rs

```diff
@@ -108,8 +108,10 @@ fn test_cname_loop() {
         &RData::CNAME(CNAME(Name::from_str("foo.example.com.").unwrap()))
     );
 
-    let additionals = lookup.additionals().expect("Should be additional records");
-    let additionals: Vec<&Record> = additionals.iter().collect();
+    let additionals: Vec<&Record> = lookup
+        .additionals()
+        .expect("Should be additional records")
+        .collect();
```

Comment:
Suggest writing this as `.collect::<Vec<_>>()`, avoiding the need for a type annotation (which is syntactically further away from the `collect()` call).